routes/cube_routes.js

```diff
@@ -1569,19 +1569,12 @@ router.get('/draft/:id', function(req, res) {
           });
         });
       });
-      draft.picks.forEach(function(card, index) {
-        if (Array.isArray(card)) {
-          card.forEach(function(item, index2) {
-            if (item) {
-              console.log(item);
-              item.details = carddb.cardFromId(item).cardID;
-              item.details.display_image = util.getCardImageURL(item);
-            }
-          });
-        } else {
+      // TODO this only handles the user picks (item 0 of draft picks), so custom images won't work with bot picks.
+      draft.picks[0].forEach(function(col, index) {
```

Comment:
This is a nitpick, but I would love if we could move towards using for..of instead of forEach as it's cleaner

Reply:
ok sure - this wasn't fresh code i was just making what was already there work.